community/ndp/kernelextension/src/main/java/org/neo4j/ext/NDPKernelExtension.java

```diff
@@ -94,7 +94,8 @@ public Lifecycle newKernelExtension( Dependencies dependencies ) throws Throwabl
         final Config config = dependencies.config();
         final GraphDatabaseService gdb = dependencies.db();
         final GraphDatabaseAPI api = (GraphDatabaseAPI) gdb;
-        final Log log = dependencies.logService().getInternalLog( Sessions.class );
+        final LogService logging = dependencies.logService();
+        final Log log = logging.getInternalLog( Sessions.class );
```

Comment:
Perhaps consider if NDP should write anything to the store internal log at all? The internal log is related to the store, and not really for the server (imagine what happens when there are multiple stores, for example).